src/please.go

```diff
@@ -167,8 +167,9 @@ var opts struct {
 		InTempDir  bool   `long:"in_tmp_dir" description:"Runs in a temp directory, setting env variables and copying in runtime data similar to tests."`
 		EntryPoint string `long:"entry_point" short:"e" description:"The entry point of the target to use." default:""`
 		Parallel   struct {
-			NumTasks       int  `short:"n" long:"num_tasks" default:"10" description:"Maximum number of subtasks to run in parallel"`
-			Quiet          bool `short:"q" long:"quiet" description:"Suppress output from successful subprocesses."`
+			NumTasks       int                `short:"n" long:"num_tasks" default:"10" description:"Maximum number of subtasks to run in parallel"`
+			Quiet          bool               `short:"q" long:"quiet" description:"Suppress output from successful subprocesses."`
```

Comment:
Can we mention that this is deprecated here too? Also we should raise an issue to remove it and add it to the v17 milestone.

Reply:
Sure. I might go ahead as well and clarify the removal version on the warning below.